doc/developer-guides/watchdog-hld.rst

```diff
@@ -0,0 +1,97 @@
+.. _watchdog-hld:
+
+Watchdog Virtualization in Device Model
+#######################################
+
+This document describes the watchdog virtualization implementation in
+ACRN device model.
+
+Overview
+********
+
+A watchdog is an important hardware component in embedded systems, used
+to monitor the system's running status, and resets the processor if the
+software crashes. In general,  hardware watchdogs rely on a piece of
+software running on the machine which must "kick" the watchdog device
+regularly, say every 10 seconds. If the watchdog doesn't get "kicked"
+after 60 seconds, for example, then the watchdog device asserts the
+RESET line which results in a hard reboot.
+
+For ACRN we emulate the watchdog hardware in the Intel 6300ESB chipset
+as a PCI device called 6300ESB watchdog and is added into the Device
+Model following follow the PCI device framework. The following
+:numref:`watchdog-device` shows the watchdog device workflow:
+
+.. figure:: images/watchdog-image2.png
+   :align: center
+   :width: 900px
+   :name: watchdog-device
+
+   Watchdog device flow
+
+The DM in the service OS (SOS) treats the watchdog as a passive device.
+It receives read/write commands from the watchdog driver, does the
+actions, and returns.  In ACRN, the commands are from user OS (UOS)
```

Comment:
Same comment about using capital U for User OS

Reply:
@dbkinder , could you change "user OS" --> "User OS"